CONTRIBUTING.md

```diff
@@ -37,6 +37,15 @@ Install PHP-CS-Fixer first: `composer -d tools install`
 
 Then run the following command trying to auto-fix the issues: `./tools/vendor/bin/php-cs-fixer fix`
 
+For JavaScript code inside `assets/` directory, we provided ESLint setup and configuration for linting.
```

Comment:
```suggestion
For JavaScript code inside the `assets/` directory, we use an ESLint setup and configuration for linting.
```

Reply:
applied the first one, the second change make it feel weird so I ended up rewording the entire half